File: Ios/denarii/denarii/Wallet.swift
//
//  Wallet.swift
//  denarii
//
//  Created by Andrew Katson on 5/28/23.
//

import Foundation

struct Wallet : Codable {
    var responseCode: Int = 0
    var responseCodeText: String = ""
    var response: WalletDetails = WalletDetails()
}

File: Android/app/src/main/java/com/denarii/android/user/Wallet.java
package com.denarii.android.user;

import com.google.gson.annotations.SerializedName;

public class Wallet {

    @SerializedName("response_code")
    public int responseCode;

    @SerializedName("response_code_text")
    public String responseCodeText;

    @SerializedName("response")
    public WalletDetails response;
}
